OpenRA.Mods.Common/Widgets/Selection.cs

```diff
@@ -0,0 +1,246 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2020 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using OpenRA.Graphics;
+using OpenRA.Primitives;
+using OpenRA.Widgets;
```

Comment:
Unnecessary using directive.

Reply:
Left over from pulling out the Selection.cs code from LabelWithSelection.cs. Will fix.